extensions/wikia/ARecoveryEngine/ARecoveryEngineHooks.class.php

```diff
@@ -5,6 +5,7 @@ class ARecoveryEngineHooks {
 	public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgARecoveryEngineCustomLog';
 		$vars[] = 'wgAdDriverSourcePointRecoveryCountries';
+		$vars[] = 'wgAdDriverSourcePointMMSCountries';
```

Comment:
I know it's pedantic but I'd go with `wgAdDriverSourcePointMmsCountries` ;) Despite the fact that MMS is an abbreviation :japanese_ogre: